extensions/src/ACRE2Mumble/MumbleCallbacks_static.cpp

```diff
@@ -29,3 +29,29 @@ mumble_version_t mumble_getAPIVersion() {
 mumble_version_t mumble_getVersion() {
     return mumble_version_t{ACRE_VERSION_MAJOR, ACRE_VERSION_MINOR, ACRE_VERSION_SUBMINOR};
 }
+
+struct MumbleStringWrapper mumble_getAuthor() {
+    static const char* author = "ACRE2 developers";
```

Comment:
Should return `ACRE_TEAM_URL` as in TeamSpeak.
https://github.com/IDI-Systems/acre2/blob/a0572a4dd4fe871b8b896e544300605f4cd0d87c/extensions/src/ACRE2TS/TSCallbacks_init.cpp#L43
or `ACRE_TEAM` if more fitting.

Macro source: https://github.com/IDI-Systems/acre2/blob/74b5b20fa028da206543e8100057019cae083f77/extensions/src/ACRE2Shared/_CONSTANTS.h